sass/_FieldTable.scss

```diff
@@ -65,13 +66,15 @@
 
 .coveo-field-table-toggle-icon-up {
   position: absolute;
-  bottom: 10px;
+  bottom: 35px;
   right: 10px;
   margin: 0;
+  @include transition(opacity 0s);
   @include retina-sprite($arrow-up-group);
-  display: none;
+  opacity: 0;
```

Comment:
Don't forget that `opacity: 0` vs `display:none` makes the component clickable even those it is not visible.